src/pages/resources/recipes/file-operation/index.md

```diff
@@ -0,0 +1,301 @@
+
+## Concepts
+
+### Sandbox and other locations
+UXP, by default, only allows access to certain locations in the user's file system. These locations are referred to as the sandbox. 
+
+<InlineAlert variant="info" slots="header, text1, text2"/>
+
+Scripts and plugins
+
+**In plugins**, a sandbox is typically the plugin's folder, plus a temporary and data folder. The plugin's location is read-only. Whereas, the temporary and data folders are provided to you to store transitory information. Note that the data stored here can get accidentally erased and should not be considered permanent.
+
+**In scripts**, lorem ipsum. // TODO What about sandbox locations in scripts?
+
+
+However, we understand that there are circumstances when you would like to have access to other locations as well. Accessing such locations, too, is possible in UXP but you will need to seek permission first.
+
+
+### Manifest permission
+For file system access you require permission for the `localFileSystem` module.
+
+<InlineAlert variant="info" slots="header, text1, text2"/>
+
+Scripts and plugins
+
+**In scripts**, these permissions are fixed to '' and you cannot alter them //(TODO Add link to details) and ignore the manifest details below. 
+
+**In plugins**, you should seek permission for `localFileSystem` in your manifest.<br></br> IMPORTANT: Be sure you know about the [manifest permissions](../../../plugins/concepts/manifest/#permissionsdefinition) module before you proceed.
+
+
+Let's understand the manifest settings a bit more in detail.
+```json
+/* manifest.json */
+{  
+    "requiredPermissions": {
+        "localFileSystem": "plugin"
+    }
+}
+```
+Allowed values for `localFileSystem` are:
+- `plugin`: Default value. Provides access to the sandbox.
+- `request`: Let's you request the user to select a folder location of their choice.
+- `fullAccess`: Provides full access to the user file system
+//TODO Q: Does fullAccess file system include system folders?
+
+*Protip*: Make sure you pick the most accurate permission for your use case because in the future we may ask users to provide their consent based on it. You may find 'fullAccess' to be the least restrictive and hence the safest to pick, but a user may not be comfortable giving full access to their system unless it's absolutely necessary and might deny the installation of your plugin. 
+
+### Schemas
+UXP provides a shorthand representation of these locations via schemas.
+
+For sandbox, you can use `plugin:/`, `plugin-data:/`and `plugin-temp:/`. // TODO Q: Do these work in scripts? <br></br>
+And, for other locations, use `file:/`.
+
+```html
+<img src="plugin:/sample.png" />
+<img src="file:/Users/user/Downloads/sample.png" /> <!-- update the path based on your system -->
+```
+
+## APIs
+You have two options to access the file system - `LocalFileSytem` and `FS` module.
+// TODO Q: How should a developer choose between the FS module and localfilesystem? What special use cases are supported by them?
```

Comment:
@kareemmyk  How should a developer choose between the FS module and localfilesystem? What special use cases are supported by them?

Reply:
So is it safe to say its developers preference which API they pick because both are equivalent in terms of the functionality they provide?

Reply:
FS is more like a single operation while localFileSystem is more like doing multiple operations on the same file